massa-module-cache/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "massa_module_cache"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+schnellru = "0.2.0"
+rocksdb = "0.20"
+serial_test = "1.0.0"
+rand = "0.8.5"
+num_enum = "0.5"
+nom = "7.1"
+displaydoc = "0.2"
+thiserror = "1.0"
+anyhow = "1.0"
+tracing = "0.1"
+# massa
+massa_hash = { path = "../massa-hash" }
+massa_models = { path = "../massa-models" }
+massa_serialization = { path = "../massa-serialization" }
+massa-sc-runtime = { git = "https://github.com/massalabs/massa-sc-runtime", branch = "caching-followup", features = [
+    "testing",
+] }
+
+[dev-dependencies]
+tempfile = "3.3"
+
+[features]
+testing = []
```

Comment:
Feature testing for massa-sc-runtime should be included only here and not in the dependencies in normal case

Reply:
Done